test/integration/webhookgithub_test.go

```diff
@@ -20,8 +20,6 @@ import (
 	testutil "github.com/openshift/origin/test/util"
 )
 
-const whPrefix = "/osapi/v1beta1/buildConfigHooks/"
```

Comment:
`TestWebhookGithubPing` needs fixing as well... not sure where the whPrefix it's using is coming from